tools/createDist.js

```diff
@@ -24,7 +24,7 @@ const createDist = async () => {
   }, dependencies);
 
   // Removes .git from postinstall dependencies
-  scripts.postinstall = 'husky install';
+  delete scripts.postinstall;
```

Comment:
This code works and is fine - could it be written using object destructuring to avoid mutating objects though?

Reply:
Oh yeah - ok, I've put it back to how it worked before to not mutate. Thanks!